src/components/verification/index.tsx

```diff
@@ -0,0 +1,192 @@
+"use client";
+
+import CommonInput from "@/components/common/inputs/input";
+import CommonInputFile from "@/components/common/inputs/fileInput";
+import CommonLabel from "../common/containers/label";
+import React from "react";
+import { useTranslation } from "react-i18next";
+import SubmitButton from "../common/containers/submit-button";
+import { useState, useRef } from "react";
+import { useFormik } from "formik";
+import SaveBooking from "../booking/partial/submitBooking";
+import { Booking, BookingFormData, Boat } from "@/models/models";
+import { useRouter } from "next/navigation";
+
+const FormWrapper = ({ children }: { children: React.ReactNode }) => {
+  return (
+    <div className="relative md:w-[48%] w-full mb-6 md:mb-0">{children}</div>
+  );
+};
+
+export default function VerificationForm({
+  bookingDetails,
+  boatInfo,
+  bookingId,
+}: {
+  bookingDetails: Booking,
+  boatInfo: Boat,
+  bookingId: string
+}) {
+  const { t } = useTranslation();
+  const saveModalRef = useRef<{ start: () => void }>(null);
+  const router = useRouter();
+
+  const [data, setData] = useState<Partial<BookingFormData>>({
+    "First Name": "",
+    "Last Name": "",
+    "ID Number": "",
+    documentType: "National ID",
+    ID_Front_Picture: {} as File,
+    ID_Back_Picture: {} as File,
+  });
+
+  const submitVerificationForm = async () => {
+    saveModalRef.current?.start();
+    console.log(">>>>>>>>Info", data)
+  };
+
+  const formik = useFormik({
+    initialValues: data,
+    onSubmit: () => {
+      submitVerificationForm();
+    },
+  });
+
+  if (!data || !formik) {
+    return;
+  }
+
+  return (
+    <>
+      <SaveBooking
+        ref={saveModalRef}
+        formData={data as BookingFormData}
+        setFormData={setData}
+        boat={boatInfo}
+        booking={data as unknown as Booking}
+        onSuccess={() => router.replace("/success")}
+        bookingId={bookingId}
+        steps={["validateFront", "validateBack", "uploadFrontIdImage", "uploadBackIdImage", "pay", "saveData", "notifyCustomer"]}
```

Comment:
dont use saveData (you are having issues right? 
Save data create a slot an other stuff

Lets create a different step for it with a different name